### PR TITLE
Added a fuction/method setTerminalsize in Exec.ExecBuilber

### PR DESCRIPTION
**Pull request by sriganeshres**
I have added a function in Exec.ExecBuilder class which helps to set the Terminal size.

### CODE REVIEW

**Comment by sriganeshres** [comment]
related to #3641

File: util/src/main/java/io/kubernetes/client/Exec.java [hunk 1]
```diff
@@ -92,7 +92,8 @@ public void setApiClient(ApiClient apiClient) {
   }
 
   /**
-   * Get a {@link Consumer<Throwable>} that will be accepted if there is any unhandled exception
+   * Get a {@link Consumer<Throwable>} that will be accepted if there is any
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/Exec.java:95]: Please revert all formatting changes here and below.

**Comment by brendandburns** [comment]
Thanks for starting this. Two notes:
First, please don't include any unrelated formatting changes, it makes it harder to review the PR, you can use `git diff HEAD~1` to visualize any differences in your commit before you send the PR.

Second, this change isn't quite correct, you need to add a utility method to the `Exec` class, not to the builder. The builder is what sets up an `Exec` instance, but the terminal resize can only happen once the exec call starts. If the method is added to the `Exec` class, then someone who starts a call to `Exec` can resize the stream once the connection is established.

**Comment by sriganeshres** [comment]
Sorry @brendandburns for very very late response. But can you guide me more than just that im facing a bit of issue here while trying to maintain that it should be able to be run after execute(). Thanks for the support till now.

**Pull request closed without merging**